go.mod

```diff
@@ -137,7 +137,7 @@ require (
 	github.com/shirou/w32 v0.0.0-20160930032740-bb4de0191aa4
 	github.com/shuLhan/go-bindata v3.4.0+incompatible
 	github.com/soniah/gosnmp v1.26.0
-	github.com/spf13/afero v1.2.2
+	github.com/spf13/afero v1.4.1
```

Comment:
Is this an accident? Why these updates to `go.mod`?

Reply:
For afero, AIX support was released in `v1.4.1`. I bumped up the version only as far as necessary. (https://github.com/spf13/afero/pull/210)

I'm not sure if the `golang.org/x/text` is a result of that or why that got updated. I'll look into it.

The viper change was Datadog/viper#15

Reply:
The `golang.org/x/text` version bump is a result of the `afero` bump.